### DISCUSSION TITLE
How to search previews?

### DISCUSSION THREAD

**Discussion by Hex4dec** [junegunn/fzf#3953] [Q&A] [answered]
I'm trying to make a makeshift manpage searcher and one thing I would like is to search the manpage contents in combination with the names. I'm currently using `man -k . | awk '{print $1.$2}' | fzf --preview 'man {}'`.

**Comment by junegunn** [comment]
You can't. As the name implies, `preview` feature is for previewing the content of the current item. If we want to make the preview output searchable, fzf should execute the preview command with every item in the input list, and it's going to be incredibly slow.

**Reply by Hex4dec** [reply]
I mean, it should at least be allowed..

**Reply by junegunn** [reply, marked as answer]
No. It's just beyond the scope of the feature.
